jsonExplorer.js

```diff
@@ -177,22 +177,23 @@ function modify(method, value) {
                     break;
 
                 case 'IGNOREVALUE':
-                    value = {
-                        value: value,
-                        ignore: true
-                    };
+                    result = null;
```

Comment:
nun wird NULL in der Funktion übergeben und auch NULL in das Attribute geschrieben oder sehe ich das falsch?

Reply:
es wird nur geschrieben !== null 

https://github.com/HGlab01/ioBroker-jsonExplorer/pull/105/files#diff-044589227090649f2c5249112b0d8e5231e999a759b6de9e725eada8c2d2a5b4R319-R329

Reply:
das geht nicht; ich schreibe öfters bewusst den Wert NULL, dass muss erhalten bleiben

Reply:
ok dan muessen wir das anders angehen, ich hatte es erstmal so aufgenommen da ich die werte eventuell noch verarbeiten wollen.

ich werde die routine nochmal überdenken und an einer anderen stellen einbauen, melde mich wen fertig

Reply:
Habe die logic abgelehnter, anstatt in der modify routine wird es jetzt im state create erledigt.
Sobald er beim durchlaufen der array (fuer modify) das attribute "ignoreValue" trifft wird eine variable gesetzt, diese bestimmt dan spatter ob der wert wohl oder nicht geschrieben wird.

Damit sollten wir mein requirement, möglich des wert schreiben zu ignorieren, plus dein anlegen (wohl NULL werte schreiben können) beide erfüllt haben :)